tsconfig.json

```diff
@@ -2,18 +2,19 @@
   "$schema": "https://json.schemastore.org/tsconfig",
   "extends": "eslint-config-upleveled/tsconfig.base.json",
   "compilerOptions": {
-    // Based on expo/tsconfig.base.json
-    // https://github.com/expo/expo/blob/main/packages/expo/tsconfig.base.json
-    "jsx": "react-native",
-    "target": "ESNext"
+    "plugins": [
+      {
+        "name": "next"
+      }
+    ]
```

Comment:
this change seems wrong, do we have something wrong in our framework detection in `eslint-config-upleveled`?

https://github.com/upleveled/eslint-config-upleveled/blob/6fafffc0012b70e1a54aabdfca000a1e083e1ce4/bin/install.js#L25-L34

looks like it - can you do a PR?

Reply:
Was fixed in this [ESLint PR](https://github.com/upleveled/eslint-config-upleveled/pull/398), and updated with this commit in this PR [f2c33e6](https://github.com/upleveled/expo-example-spring-2024-atvie/pull/4/commits/f2c33e6e7317cb16183dd16a79f1845338d6cfbe)